project.js

```diff
@@ -2,59 +2,17 @@ const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
 function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        rl.question(questionText, resolve);
-    });
-}
-
-function randomInteger(min, max) {
-    let range = max - min + 1;
-    return min + Math.floor(Math.random() * range);
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
 }
 
 start();
 
-function getRandomInt(max) {
-    return Math.floor(Math.random() * Math.floor(max));
-}
-
 async function start() {
-    console.log("Let's play a game where you (human) make up a number between 1 and 100 and I (computer) try to guess it.")
-    let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-    console.log('You entered: ' + secretNumber);
-
-    let guess = getRandomInt(100)
-    let userResponse = await ask("Is " + guess + " your number? ")
-
-    let maxNumber = 100
-    let minNumber = 1
-
-    while (userResponse !== "Y" || "y" !== userResponse) {
-
-        if (guess > secretNumber) {
```

Comment:
Imagine you don't have access to `secretNumber` how could we write this so that the person specifically tells the computer that it's higher or lower.